pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstance.java

```diff
@@ -22,9 +22,11 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
+import org.apache.pulsar.functions.api.Context;
```

Comment:
what are these two imports for?

Reply:
was for debugging but not needed anymore